packages/pagination/package-lock.json

```diff
@@ -3,3978 +3,6 @@
   "version": "4.1.1",
   "lockfileVersion": 2,
   "requires": true,
-  "packages": {
```

Comment:
You've deleted all packages from the package lock 😬

Reply:
I must've done that in a previous commit. Repairing this rn

Reply:
I think they disappear when I use npm install commands